_posts/2023-07-07-announcing-ballerina-2201.7.0-swan-lake-update-7.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: blog-post
+title: Announcing Ballerina 2201.7.0 (Swan Lake Update 7)
+author: Ballerina Team
+published-date: 7 July 2023
+status: Published
+socialmediaimage: Ballerina-Swan-Lake-GA-Release-banner-02-with-button.png
+permalink: /posts/2023-07-07-announcing-ballerina-2201.7.0-swan-lake-update-7/
+---
+
+<style>.cBlogContent p{white-space: break-spaces !important;}</style>
+
+We are excited to announce the [Ballerina 2201.7.0 (Swan Lake Update 7)](https://ballerina.io/downloads/) release, which adds many new features and improvements to the Ballerina programming language. 
+
+The two main highlights of this release are the support for generating GraalVM native executables and adding aggregation and grouping capabilities to Ballerina query expressions and query actions, which are described below.
```

Comment:
Is there a better way to say this? two highlights were mentioned in two different way.
i.e first one started as  `support for ....` and second one started as `adding...`

Reply:
The `support for` phrase applies to both phrases starting with `generating` and `adding` here.